nostrpostrlib/src/main/java/nostr/postr/RelayPool.kt

```diff
@@ -10,12 +10,26 @@ object RelayPool: Relay.Listener {
     private val listeners = HashSet<Listener>()
     private val eventIds = HashSet<String>()
 
-    init {
-        Constants.defaultRelays.forEach { addRelay(it) }
+
+    fun loadRelays(relayList: List<Relay>? = null){
+        if (!relayList.isNullOrEmpty()){
+            relayList.forEach { addRelay(it) }
+        } else {
+            Constants.defaultRelays.forEach { addRelay(it) }
+        }
+    }
+
+    fun sendFilter(subscriptionId: String) {
+        relays.forEach { it.connect(subscriptionId = subscriptionId) }
```

Comment:
Still, Client.requestAndWatch() calls RelayPool.sendFilter() which callse Relay.connect().

Why not have Client.requestAndWatch() call RelayPool.requestAndWatch() which calls Relay.requestAndWatch()?

Also Relay.connect(subscriptionId: String) is wrong. The Relay can connect without subscribing. It was an error on my part to mix these two and I'm sorry.

As an **additional feature** it might be useful to have a way to say "get this stuff from these relays" in one command as else the relay isn't in the right state the the API user has to wait for connect but for any bigger app that uses more than one subscription, monitoring connection state should be done by the API user.